iFixit/react-commerce
# Social gallery tiles link only on the @author text

closes #1737 

### QA

1. Visit Vercel preview
2. Verify that links are now only on the `@author` text and that the linked page opens in a new tab
